src/main/groovy/shiro/ShiroGrailsPlugin.groovy

```diff
@@ -133,7 +135,15 @@ Enables Grails applications to take advantage of the Apache Shiro security layer
             }
 
             // Default remember-me manager.
-            shiroRememberMeManager(CookieRememberMeManager)
+            def cipherKeyFromConfig = grailsApplication.config.getProperty('security.shiro.rememberMe.cipherKey')
```

Comment:
What type are you expecting here?
getProperty allows you to specify the type and the default value